apps/meteor/client/views/home/cards/MobileAppsCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button } from '@rocket.chat/fuselage';
+import { useTranslation } from '@rocket.chat/ui-contexts';
+import React, { ReactElement } from 'react';
+
+import CardBody from '../../../components/Card/Body';
+import Card from '../../../components/Card/Card';
+import CardFooterWrapper from '../../../components/Card/CardFooterWrapper';
+import CardFooter from '../../../components/Card/Footer';
+import CardTitle from '../../../components/Card/Title';
+import ExternalLink from '../../../components/ExternalLink';
+
+const GOOGLE_PLAY_URL = 'https://google.com';
+const APP_STORE_URL = 'https://google.com';
```

Comment:
We need to update these values right?

Reply:
Yes, this will come in another PR

Reply:
Actually, we just got the links so I added them too.